ownCloud/Viewer/DisplayViewController.swift

```diff
@@ -20,6 +20,7 @@ import UIKit
 import ownCloudSDK
 
 struct DisplayViewConfiguration {
+	weak var rootItem: OCItem!
```

Comment:
I think this, too, is a left-over addition and can be removed. FWIW it doesn't seem to be needed/used anywhere.